examples/ts-jest/tsconfig.json

```
@@ -0,0 +1,23 @@
+{
+  "compilerOptions": {
+    "strict": true,
+    "noEmitOnError": true,
+    "suppressImplicitAnyIndexErrors": true,
+    "target": "ES5",
+    "plugins": [
+      {
+        "transform": "ts-transformer-keys/transformer"
+      }
+    ]
```

`plugins` field does not make sense because `ttypescript` is not used in this example, right?

```suggestion
    "target": "ES5"
```

---

You're right! It is not needed, I copied over the ttscrypt example as starting point and did not realize.